manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
+PackageVersion: 6.2.2.0

```

Comment:
PackageVersion needs to match the version you find in the control panel.
![image](https://user-images.githubusercontent.com/52826317/124193201-b7628900-dac6-11eb-8f38-4c5084b4e954.png)

```suggestion
PackageVersion: 6.2.2
```